clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -125,10 +130,29 @@ export const ErrorMessage = ({
   );
 };
 
+const ClearIndicator = () => null;
+
 export interface Option {
   value: string;
   label: string;
+  description?: string;
 }
+
+const CustomOption: React.FC<
```

Comment:
The `SelectComponentsConfig<Option, boolean, GroupBase<Option>>` type for `components` added below caused the linter to start throwing errors because it requires a `ReactElement` type. I needed to refactor it out into a dedicated component to fix those errors.

Reply:
I also decided to not make `CustomOption` component a prop that can be passed down to keep things simple. It we start requiring more custom option components it can be updated in the future to accommodate that

Reply:
can we leave a comment about what this custom option is? is it that we conditionally render a description if one exists?